apps/web-giddh/src/app/invoice/preview/invoice.preview.component.scss

```diff
@@ -653,7 +653,7 @@ span.icon-export-icon {
         background-color: $white;
         z-index: -1;
     }
-    .displayBlock {
+    .d-block {
```

Comment:
Is this class required if we have this class in global styles?

Reply:
not required, deleted @prateek-giddh

Reply:
not required, deleted